scripts/build-icons.js

```diff
@@ -140,7 +141,7 @@ for (const icon of whitelist) {
     let slug = (overrides[icon] || {})[type] || icon
     const filepath = `tmp/${name}${type}.svg`
     const folder = `${assetsFolder}/${name}${type}.imageset`
-    run(`curl -sSL https://raw.githubusercontent.com/instructure/instructure-ui/master/packages/ui-icons/svg/${type}/${slug}.svg > ${filepath}`)
+    run(`curl -sSL https://raw.githubusercontent.com/instructure/instructure-ui/c1c7d673/packages/ui-icons/svg/${type}/${slug}.svg > ${filepath}`)
```

Comment:
would a little comment here about why we are using this commit be helpful for future developers

Reply:
I'm not sure what I'd say other than pin to a version instead of always latest, which I'm not sure is particularly helpful

Reply:
I thought we were pinning b/c of padding problems in master? No?